client/web/compose/src/components/ModuleFields/Configurator/Select.vue

```diff
@@ -108,21 +108,20 @@
       </b-form-group>
 
       <b-form-group
-        v-if="f.isMulti"
+        :label="$t('kind.select.optionType.label')"
       >
-        <label class="d-block">{{ $t('kind.select.optionType.label') }}</label>
         <b-form-radio-group
           v-model="f.options.selectType"
           :options="selectOptions"
           stacked
           @change="onUpdateIsUniqueMultiValue"
         />
+
         <b-form-checkbox
-          v-if="f.options.selectType !== 'multiple'"
+          v-if="shouldAllowDuplicates"
```

Comment:
This checkbox needs to be moved out of the form group, and same for User and Record selector